build.gradle.kts

```diff
@@ -42,7 +42,6 @@ buildscript {
         }
     }
     dependencies {
-        classpath(libs.sqldelight.plugin)
         classpath("com.mikepenz.aboutlibraries.plugin:aboutlibraries-plugin:11.1.3")
```

Comment:
move this into the toml file :)

Reply:
[toml file, classpath and format](https://github.com/Mindera/Android-KMP-Template/pull/9/commits/4f9fcd593cceb2e74ff2babc507f1eb8b605f576)